content/pages/publish/form-dataset.json

```diff
@@ -28,7 +28,8 @@
       "label": "Sample file",
       "placeholder": "e.g. https://file.com/samplefile.json",
       "help": "Please enter the URL to a sample of your data set file and click \"ADD FILE\" to validate the data. This file should reveal the data structure of your data set, e.g. by including the header and one line of a CSV file. This file URL will be publicly available after publishing.",
-      "type": "files"
+      "type": "files",
+      "required": true
```

Comment:
question(blocking): why was `required` added?

Reply:
Because ddo validation fails if it's not there. It was always required, was just missing the proper visual representation.